packages/react/src/components/Storage/FileUploader/Previewer/index.tsx

```diff
@@ -60,13 +68,9 @@ export function Previewer({
         </UploadDropZone>
         <Text className={ComponentClassNames.FileUploaderPreviewerText}>
           {isSuccessful ? (
-            <>
-              {uploadedFilesLength()} {translate('files uploaded')}
-            </>
+            <>{`${getUploadedFilesLength()} ${translate('files uploaded')}`}</>
```

Comment:
Still think you should be able to remove the fragments here, but maybe i'm wrong?

Reply:
I get an error when I remove the `<></>` fragments. ` "`Expected identifier but found "`${"`